### DISCUSSION TITLE
Postgres with SSL

### DISCUSSION THREAD

**Discussion by devtribe** [mend/renovate-ce-ee#601] [Q&A]
Hi,

our Postgres Instance requires SSL.

It is possible with a connection string like `postgresql://11.20.50.50:5432/renovate?sslmode=require`, but i see no possiblity to use  this for renovate. Did i miss an env variable?

**Comment by rarkins** [comment]
You didn't mention if you've seen https://github.com/mend/renovate-ce-ee/blob/51a8b553121fa5b6dd7c861dba46d9b585eaeaf4/docs/configure-postgres-db.md?plain=1#L8

Does SSL with raw IPs work for Postgres?
